src/main/java/org/jenkinsci/plugins/docker/commons/fingerprint/DockerFingerprints.java

```diff
@@ -23,6 +23,15 @@ private static String trim(String imageId) {
         }
         return imageId.substring(0, 32);
     }
+    
+    /**
+     * Gets a fingerprint hash for Docker image.
+     * @param imageId Docker image ID.
+     * @return 32-symbol fingerprint hash
+     */
+    /*package*/ static @Nonnull String getImageFingerprintHash(@Nonnull String imageId) {
+        return trim(imageId);
```

Comment:
<code>trim()</code> has a weird name for public methods. BTW seems it makes sense to have a public method for external users, who may want to convert imageId to hash w/o retrieving <code>Fingerprint</code>s. It's a useful approach due to performance reasons.